docs/source/app_developers_guide/docker.rst

```diff
@@ -2,696 +2,672 @@
 
-* Runs a single XO transaction processor
-* Handles transactions of the XO transaction family
-* Hostname: ``xo-tp-python``
-* Container name: ``sawtooth-xo-tp-python-default``
+    .. code-block:: console
 
-Log into this container by running this command from the host computer's
-terminal window:
+       root@21bb5af673a8:/# ps --pid 1 fw
+       PID TTY      STAT   TIME COMMAND
+        1 ?        Ss     0:00 bash -c sawadm keygen && sawtooth keygen my_key
+       && sawset genesis -k /root/.sawtooth/keys/my_key.priv && sawadm genesis
+       config-genesis.batch && sawtooth-validator -vv --endpoint
 
-.. code-block:: console
 
-  $ docker exec -it sawtooth-xo-tp-python-default bash
+Step 9: Examine Sawtooth Log Files
+==================================
 
-To see which components are running, run this command from the container:
+As described above, you can display Sawtooth log messages by using the
+``docker logs`` command from your host system:
 
 .. code-block:: console
 
-  $ ps --pid 1 fw
-    PID TTY      STAT   TIME COMMAND
-    1 ?        Ssl    0:00 /usr/bin/python3 /usr/bin/xo-tp-python -vv tcp://validator:4004
-
+   # docker logs {OPTIONS} {ContainerName}
 
-Viewing Log Files
-=================
+In each container, the Sawtooth log files for that component are stored in the
+directory ``/var/log/sawtooth``. Each component (validator, REST API, and
+transaction processors) has both a debug log and an error log.
 
-You can view the log files for any running Docker container using the
-``docker logs`` command.
-Replace ``{CONTAINER}`` with the name of one of the Sawtooth Docker containers,
-such as ``sawtooth-validator-default``.
+For example, the validator container has these log files:
 
 .. code-block:: console
 
-  $ docker logs {CONTAINER}
+   $ ls -1 /var/log/sawtooth
+   validator-debug.log
+   validator-error.log
 
+The IntegerKey container has these log files:
 
-Configuring the List of Transaction Families
-============================================
-
-Next, tell the validator or validator network to accept
-transactions from the IntegerKey and Settings transaction families.
+.. code-block:: console
 
-Sawtooth provides a :doc:`Settings transaction family
-<../transaction_family_specifications/settings_transaction_family>` that stores
-on-chain configuration settings, along with a Settings family transaction
-processor written in Python.
+   $ ls -1 /var/log/sawtooth
+   intkey-ae98c3726f9743c4-debug.log
+   intkey-ae98c3726f9743c4-error.log
 
-One of the on-chain settings is the list of supported transaction families.
-In the example below, a JSON array is submitted to the ``sawset``
-command, which creates and submits a batch of transactions containing the
-configuration change.
+.. note::
 
-The submitted JSON array tells the validator or validator network to accept
-transactions of the following types:
+   By convention, the transaction processors use a string to make the log file
```

Comment:
I would change "string" to "random string", since they are all strings

Reply:
Fixed